tgui/packages/tgui/interfaces/CharacterPreferences/index.tsx

```diff
@@ -0,0 +1,178 @@
+              </Box>
+            </Flex.Item>
+            <Flex.Item ml="5px">
+              <Section fill>
+                <ByondUi
+                  params={{
+                    id: data.preview,
+                    type: 'map',
+                  }}
+                  style={{
+                    width: '80px',
+                    height: '80px',
+                  }}
+                />
+                <Box textAlign="center" mt="5px">
+                  <Button icon="chevron-left" onClick={() => act('rotate-clockwise')} />
+                  <Button icon="chevron-right" onClick={() => act('rotate-counter-clockwise')} />
+                </Box>
+              </Section>
+            </Flex.Item>
+          </Flex>
+        ) : (
+          <Fragment>
+            {menu === CharacterPreferencesTabKeys.GameSettings && <GameSettingsTab />}
+            {menu === CharacterPreferencesTabKeys.Saves && <SavesTab />}
+          </Fragment>
+        )}
+
+        <Section mt="5px">
+          <Button.Confirm content="Reset All" onClick={() => act('reset')} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const SavesAndProfile = (_props, context) => {
+  const { act, data } = useBackend<CharacterPreferencesData>(context);
+
+  const activeProfileIndex = data.profiles.findIndex((p) => p.active);
+
+  return (
+    <Fragment>
+      <Flex mb="5px">
+        {data.profiles.map((profile, index) => (
+          <Flex.Item key={index} ml={index === 0 ? '0' : '5px'} grow="1">
+            <Profile profile={profile} index={index} />
+          </Flex.Item>
+        ))}
+      </Flex>
+
+      <Section>
+        <LabeledList>
+          <LabeledList.Item
+            label="Profile Name"
+            buttons={
+              activeProfileIndex > -1 ? (
+                <Fragment>
+                  <Button onClick={() => act('load', { index: activeProfileIndex + 1 })}>Reload</Button> -{' '}
+                  <Button onClick={() => act('save', { index: activeProfileIndex + 1 })}>Save</Button>
+                </Fragment>
+              ) : null
+            }>
+            <Button onClick={() => act('update', { profileName: 1 })}>
+              {data.profileName ? data.profileName : <Box italic>None</Box>}
+            </Button>
+          </LabeledList.Item>
+        </LabeledList>
+        {data.profileModified ? (
+          <Fragment>
+            <Divider />
+            <NoticeBox danger mt="10px">
+              <Flex>
+                <Flex.Item align="center" mr="5px">
+                  <Icon name="exclamation-triangle" />
+                </Flex.Item>
+                <Flex.Item>
+                  <Box>You may have unsaved changes!</Box>
+                  <Box>Any unsaved changes will take effect for this round only.</Box>
+                </Flex.Item>
+              </Flex>
+            </NoticeBox>
+          </Fragment>
+        ) : null}
```

Comment:
You can avoid using `? :` by casting data to boolean with `!!data.profileModified` and using the `&&` operator.

Real `true/false` booleans are not rendered by React/Inferno, numbers are.